web/controllers/userController/index.ts

```diff
@@ -1018,18 +1025,65 @@ async function putActive(wix_id: string, active: boolean, person: Pupil | Studen
                 await dissolveMatch(matches[i], 0, person);
             }
 
-            // Step 2: Deactivate
+            // Step 2: Cancel all courses if user is student
```

Comment:
Huh, isn't that already in master? Or is the git diff wrong here?

Reply:
Hmm, this is strange... I neither changed this, nor is it displayed as a diff in vscode